cli/main.rs

```diff
@@ -533,6 +533,7 @@ async fn create_graph_and_maybe_check(
         log_checks: true,
         reload: ps.options.reload_flag(),
       },
+      ps.npm_resolver.clone(),
```

Comment:
Consider making it second to last argument (or moving it to `CheckOptions` instead)